content/sensu-go/6.4/operations/maintain-sensu/upgrade.md

```diff
@@ -11,25 +11,67 @@ menu:
     parent: maintain-sensu
 ---
 
-To upgrade to the latest version of Sensu Go, [install the latest packages][1] and restart the services.
+To upgrade to the latest version of Sensu Go:
 
-{{% notice note %}}
-**NOTE**: For systems that use `systemd`, run `sudo systemctl daemon-reload` before restarting the services.
-{{% /notice %}}
+1. Starting with your current version, read the version-specific information on this page and complete the instructions for each individual version up to the version you want to install.
+   - For example, to upgrade from 5.5.0 to 6.4.0, start with [Upgrade Sensu clusters from 5.7.0 or earlier to 5.8.0 or later][14].
 
-To restart the Sensu agent, run:
+2. [Install][1] the latest packages or Docker image.
 
+3. For systems that use `systemd`, run:
 {{< code shell >}}
-sudo service sensu-agent restart
+sudo systemctl daemon-reload
 {{< /code >}}
 
-To restart the Sensu backend, run:
+4. Restart the Sensu agent:
+{{< code shell >}}
+sudo service sensu-agent restart
+{{< /code >}}
 
+5. Restart the Sensu backend:
 {{< code shell >}}
 sudo service sensu-backend restart
 {{< /code >}}
 
-To confirm the installed version, run `sensu-agent version`, `sensu-backend version`, and `sensuctl version`.
+6. Upgrade the Sensu backend:
+{{< code shell >}}
+sensu-backend upgrade
+{{< /code >}}
+
+7. Run a single upgrade command on one your Sensu backends to migrate the cluster:
```

Comment:
steps 6 and 7 are asking the user to run the same command - are they to run this command twice?

Reply:
No, that is just a mistake I forgot to fix when I combined the upgrade instructions from a couple sections. Thank you, I'll fix it now.